lxd/apparmor/instance_lxc.go

```diff
@@ -78,6 +78,9 @@ profile "{{ .name }}" flags=(attach_disconnected,mediate_deleted) {
   # Handle /run remounts.
   mount options=(rw,nosuid,nodev,remount) -> /run/,
 
+  # Handle ramfs
```

Comment:
Maybe worth adding to the comment that ramfs is same as tmpfs for any future changes to the tmpfs rules?